opal/group.go

```diff
@@ -84,6 +89,11 @@ func resourceGroup() *schema.Resource {
 				Type:        schema.TypeInt,
 				Optional:    true,
 			},
+			"recommended_duration": {
+				Description: "The recommended duration for which the group should be requested (in minutes). Will be the default value in a request. Use -1 to set to indefinite and 0 to unset.",
```

Comment:
why do we need to set to 0 to unset? What happens if I remove the setting from the tf file? I would expect that to remove the recommended duration.